.idea/dbnavigator.xml

```diff
@@ -0,0 +1,489 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="DBNavigator.Project.DataEditorManager">
+    <record-view-column-sorting-type value="BY_INDEX" />
+    <value-preview-text-wrapping value="false" />
+    <value-preview-pinned value="false" />
+  </component>
+<<<<<<< HEAD
```

Comment:
This is rebase garbage, remove it.

Reply:
this <<<<<<< HEAD
and >>>>>>>
========34534534534

This kind of thing is not correct xml and was introduced by git rebase as conflicts that you don't resolve